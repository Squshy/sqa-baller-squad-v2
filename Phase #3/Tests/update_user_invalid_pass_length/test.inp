LOGIN
HECKINGCOOLMAN
chimmychinga
CHANGEPASSWORD
chimmychongaerinobaby