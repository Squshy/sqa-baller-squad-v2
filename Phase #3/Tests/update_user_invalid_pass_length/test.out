=========================================================
                    UPDATE PASSWORD
=========================================================
Enter your new password: chimmychongaerinobaby
---------------------------------------------------------
Your password cannot contain more than 12 characters.
=========================================================